src/assets/assetTypes.js

```diff
@@ -66,12 +68,17 @@ class AssetTypes {
    *   .catch((err) => console.log(err));
    */
   create(assetType = {}) {
+    const hasFieldFns = {
+      default: (object, key) => !!object[key],
+      organizationId: (object, key) => has(object, key)
+    };
     const requiredFields = ['description', 'label', 'organizationId'];
 
     for (let i = 0; i < requiredFields.length; i++) {
       const field = requiredFields[i];
+      const hasField = hasFieldFns[field] || hasFieldFns.default;
 
-      if (!assetType[field]) {
+      if (!hasField(assetType, field)) {
```

Comment:
Instead of having the `hasFieldFns` object and the `hasField` constant, couldn't you just do the following inside the `if` check here?
```
if (!has(assetType, field)) {
```

Reply:
Not really -- `organization_id` is the only field that is allowed to be null, the others require a non-null and non-undefined value. (`_.has` returns true if `undefined` or `null` is explicitly set on a property)

Reply:
If I've misunderstood what you're asking, lmk and I'll change it in another PR

Reply:
Oh my mistake. For some reason I thought `_.has` returns `true` as long as the value related to the key is not `undefined` so `null` would be okay